BlazarUI/app/scripts/components/module/BuildButton.jsx

```diff
@@ -0,0 +1,36 @@
+import React, {Component, PropTypes} from 'react';
+import {Button} from 'react-bootstrap';
+import Icon from '../shared/Icon.jsx';
+import $ from 'jquery';
+
+class BuildButton extends Component {
+
+  constructor(props) {
+    super(props);
+  }
+
+  build() {
+    this.props.triggerBuild();
+  }
+
+  render() {
+    if (this.props.buildTriggering) {
+      return (
+        <Button bsStyle="primary" disabled>
+          <Icon for="spinner" /> Build Now
+        </Button>
+      );
+    } else {
+      return (
+        <Button bsStyle="primary" onClick={this.build.bind(this)}> Build Now</Button>
```

Comment:
May be more of a preference, but for consistency's sake, I've been trying to keep the bindings of these types of functions in the constructor and using underscore's bindAll. It just makes the markup a little cleaner and organized IMO. (like -> https://github.com/HubSpot/Blazar/blob/c59cef468a37f71a954f18f6237b76f9bed6b7ae/BlazarUI/app/scripts/components/sidebar/BuildsSidebar.jsx#L19)